Frontend/css/index.css

```diff
@@ -34,3 +34,7 @@ body {
     text-align: center;
     border-top: 3px solid black;
 }
+
+.mg-10 {
+    margin: 10px;
```

Comment:
do you really need this? I thought bootstrap already has it

Reply:
yes

Reply:
[Bootstrap Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)

Reply:
> [Bootstrap Spacing](https://getbootstrap.com/docs/4.0/utilities/spacing/)

yes, of course